packages/ember-htmlbars/tests/helpers/each_test.js

```diff
@@ -1,617 +0,0 @@
-    items
-  });
-
-  runAppend(view);
-
-  assertHTML(view, 'onetwo');
-
-  run(function() {
-    items.shiftObject();
-    items.shiftObject();
-  });
-
-  assertHTML(view, 'Nothing');
-});
-
-QUnit.test('it can move to and from {{else}} properly when the backing array gains and looses items (#11140)', function() {
-  runDestroy(view);
-  var items = emberA(['one', 'two']);
-  view = EmberView.create({
-    template: compile('{{#each view.items as |item|}}{{item}}{{else}}Nothing{{/each}}'),
-    items
-  });
-
-  runAppend(view);
-
-  assertHTML(view, 'onetwo');
-
-  run(function() {
-    items.shiftObject();
-    items.shiftObject();
-  });
-
-  assertHTML(view, 'Nothing');
-
-  run(function() {
-    items.pushObject('three');
-    items.pushObject('four');
-  });
-
-  assertHTML(view, 'threefour');
-
-  run(function() {
-    items.shiftObject();
-    items.shiftObject();
-  });
-
-  assertHTML(view, 'Nothing');
-});
-
-QUnit.module('{{each bar as |foo|}}', {
-  setup() {
-    owner = buildOwner();
-    owner.register('view:toplevel', EmberView.extend());
-  },
-  teardown() {
-    runDestroy(owner);
-    runDestroy(view);
-    owner = view = null;
-  }
-});
-
-QUnit.test('#each accepts a name binding', function() {
-  view = EmberView.create({
-    template: compile('{{#each view.items as |item|}}{{view.title}} {{item}}{{/each}}'),
-    title: 'My Cool Each Test',
-    items: emberA([1, 2])
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'My Cool Each Test 1My Cool Each Test 2');
-});
-
-QUnit.test('#each accepts a name binding and does not change the context', function() {
-  var controller = EmberController.create({
-    name: 'bob the controller'
-  });
-  var obj = EmberObject.create({
-    name: 'henry the item'
-  });
-
-  view = EmberView.create({
-    template: compile('{{#each view.items as |item|}}{{name}}{{/each}}'),
-    title: 'My Cool Each Test',
-    items: emberA([obj]),
-    controller: controller
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'bob the controller');
-});
-
-QUnit.test('#each accepts a name binding and can display child properties', function() {
-  view = EmberView.create({
-    template: compile('{{#each view.items as |item|}}{{view.title}} {{item.name}}{{/each}}'),
-    title: 'My Cool Each Test',
-    items: emberA([{ name: 1 }, { name: 2 }])
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'My Cool Each Test 1My Cool Each Test 2');
-});
-
-QUnit.test('#each accepts \'this\' as the right hand side', function() {
-  view = EmberView.create({
-    template: compile('{{#each this as |item|}}{{view.title}} {{item.name}}{{/each}}'),
-    title: 'My Cool Each Test',
-    controller: emberA([{ name: 1 }, { name: 2 }])
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'My Cool Each Test 1My Cool Each Test 2');
-});
-
-QUnit.test('it doesn\'t assert when the morph tags have the same parent', function() {
```

Comment:
The history is that this test didn't originally have the tbody and was ensuring auto tbody inserting didn't break 'morph tags'.  When we updated to tbody we lost any meaning for this test.  Basically we've decided that relying on the browser to correct this html is no longer supported.